install-macos.sh

```diff
@@ -68,6 +70,29 @@ else
     exit 1
 fi
 
+##
+# Check Developer ID
+##
+printf "\033[34m\n* Ensuring package Developer ID matches\n\033[0m"
```

Comment:
Thanks so much for using the existing color scheme & everything!

Reply:
Of course!